gn3/api/rqtl2.py

```diff
@@ -1,47 +1,47 @@
 """ File contains endpoints for rqlt2"""
-
-import subprocess
-import os
+import shutil
+from pathlib import Path
 from flask import current_app
 from flask import jsonify
 from flask import Blueprint
 from flask import request
-
+from gn3.computations.rqtl2 import compose_rqtl2_cmd
+from gn3.computations.rqtl2 import prepare_files
+from gn3.computations.rqtl2 import validate_required_keys
+from gn3.computations.rqtl2 import write_input_file
+from gn3.computations.rqtl2 import process_qtl2_results
```

Comment:
*nitpick:* Here you could

```suggestion
from gn3.computations.rqtl2 import (compose_rqtl2_cmd,
                                    prepare_files,
                                    validate_required_keys,
                                    write_input_file,
                                    process_qtl2_results)
```

Above is easier to read and less clutter.